test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlCommentsTextHelperTests.cs

```diff
@@ -129,6 +129,7 @@ Misplaced Tab Indentation
         [InlineData("<c>DoWork</c> is a method in <c>TestClass</c>.", "`DoWork` is a method in `TestClass`.")]
         [InlineData("<code>DoWork</code> is a method in <code>\nTestClass\n</code>.", "```DoWork``` is a method in ```\nTestClass\n```.")]
         [InlineData("<para>This is a paragraph</para>.", "\r\nThis is a paragraph.")]
+        [InlineData("<para>   This is a paragraph   </para>.", "\r\nThis is a paragraph.")]
```

Comment:
please, let me know if this OK to trim whitespaces before `<para>` and after `</para>`

Reply:
I'm not sure - as long as things don't get squashed together.

For example we wouldn't want `<para> one </para><para> two </para>` becoming `onetwo`.

Reply:
I've added test case, it will humanized to `"\r\none\r\ntwo"` - I think it's OK